test/e2e/storage/persistent_volumes.go

```diff
@@ -128,7 +128,7 @@ var _ = utils.SIGDescribe("PersistentVolumes", func() {
 
 	// Testing configurations of a single a PV/PVC pair, multiple evenly paired PVs/PVCs,
 	// and multiple unevenly paired PV/PVCs
-	ginkgo.Describe("NFS", func() {
+	ginkgo.Describe("[Driver: nfs]", func() {
```

Comment:
There is no "right" or "wrong". I don't think we have anything that depends on the `[Driver: <some driver>]` format.

I personally prefer `NFS` because it is shorter.

Reply:
We have a lot of tests that skip nfs tests using `\[Driver:.nfs\]` regex but this one was not tagged properly.

https://cs.k8s.io/?q=%5C%5BDriver%3A.nfs&i=nope&files=&excludeFiles=&repos=